.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: build-docker-image
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
```

Comment:
What does this mean?

Reply:
So whenever a Pull Request is created or any commit is pushed to the PR it will run the job.